examples/query_lang/multi_credentials.json

```
@@ -0,0 +1,35 @@
+{
+  "expect_credentials": [
+    {
+      "group": "pid",
+      "format": "vc+sd-jwt",
+      "expect_meta": {
+        "vct_values": ["https://credentials.example.com/identity_credential"]
+      },
+      "expect_claims": [
+        {"path": ["given_name"]},
+        {"path": ["family_name"]},
+        {"path": ["address", "street_address"]}
+      ]
+    },
+    {
+      "group": "mdl",
+      "format": "mso_mdoc",
+      "expect_meta": {
+        "doctype_values": "org.iso.7367.1.mVR",
```

Should also be an array i guess?

```suggestion
        "doctype_values": ["org.iso.7367.1.mVR"],
```

---

Yes, thanks!